packages/wonder-blocks-dropdown/components/dropdown-launcher.js

```diff
@@ -207,6 +212,45 @@ export default class DropdownLauncher extends React.Component<Props, State> {
             </span>
         );
 
+        /*         TODO: Enforce better type check here
+        Currently this only looks for the 'icon' prop to decide
+        if an element is a IconButton or not as the 'isClassOf' method isnt present
+        */
+        if (childAnchor.props.icon) {
+            opener = (
+                <span
+                    ref={this.handleOpenerRef}
+                    disabled={!dropdownItems.length || disabled}
+                >
+                    {children(this._openDropdown)}
+                </span>
+            );
+        } else {
+            opener = (
+                <View
+                    ref={this.handleOpenerRef}
+                    disabled={!dropdownItems.length || disabled}
+                    style={{display: "inline-block"}}
+                >
+                    <span style={{float: "left", cursor: "pointer"}}>
+                        {children(this._openDropdown)}
+                    </span>
+                    <Icon
+                        icon={icons.caretDown}
+                        aria-label="Close modal"
+                        onClick={this._openDropdown}
```

Comment:
Flow threw one error here, could you check it please?

Reply:
its because Icon doesn't implement onClick

Reply:
We need to rework this anyways to use `ClickableBehavior`.